SECURITY.md

```diff
@@ -1,18 +1,38 @@
-# Reporting Security Vulnerabilities
+# Reporting security vulnerabilities
 
-Oracle values the independent security research community and believes that responsible disclosure of security vulnerabilities helps us ensure the security and privacy of all our users.
+Oracle values the independent security research community and believes that
+responsible disclosure of security vulnerabilities helps us ensure the security
+and privacy of all our users.
 
-Please do NOT raise a GitHub Issue to report a security vulnerability. If you believe you have found a security vulnerability, please submit a report to secalert_us@oracle.com preferably with a proof of concept. We provide additional information on [how to report security vulnerabilities to Oracle](https://www.oracle.com/corporate/security-practices/assurance/vulnerability/reporting.html) which includes public encryption keys for secure email.
+Please do NOT raise a GitHub Issue to report a security vulnerability. If you
+believe you have found a security vulnerability, please submit a report to
+[secalert_us@oracle.com][1] preferably with a proof of concept. Please review
+some additional information on [how to report security vulnerabilities to Oracle][2].
+We encourage people who contact Oracle Security to use email encryption using
+[our encryption key][3].
 
-We ask that you do not use other channels or contact project contributors directly. 
+We ask that you do not use other channels or contact the project maintainers
+directly.
 
-Non-vulnerability related security issues such as new great new ideas for security features are welcome on GitHub Issues. 
+Non-vulnerability related security issues including ideas for new or improved
+security features are welcome on GitHub Issues.
 
-## Security Updates, Alerts and Bulletins
+## Security updates, alerts and bulletins
 
-Security updates will be released on a regular cadence. Many of our projects will typically release security fixes in conjunction with the [Oracle Critical Patch Update](https://www.oracle.com/security-alerts/) program. Security updates are released on the Tuesday closest to the 17th day of January, April, July and October. A pre-release announcement will be published on the Thursday preceding each release. Additional information, including past advisories, is available on our [Security Alerts](https://www.oracle.com/security-alerts/) page.
+Security updates will be released on a regular cadence. Many of our projects
```

Comment:
There are two different ways to address security fixes in the WebLogic Remote Console:
1. Any Security fixes required by the WebLogic Remote Console application will be released soon after the Security vulnerability has been reported/found.
2. Fixes for the WebLogic Remote Console components that run on WebLogic Server will be addressed on a quarterly basis in conjunction with other CPU fixes.

Reply:
Yes, I agree.  We do not restrict ourselves to the quarterly cadence.